ipn/local.go

```diff
@@ -184,6 +179,118 @@ func (b *LocalBackend) SetDecompressor(fn func() (controlclient.Decompressor, er
 	b.newDecompressor = fn
 }
 
+func (b *LocalBackend) setClientStatus(newSt controlclient.Status) {
```

Comment:
add a little godoc on these methods explaining what they're called and what it does? this code's all complicated enough so let's help the next reader.